khal/khalendar/event.py

```diff
@@ -774,7 +774,7 @@ class AllDayEvent(Event):
 
     @property
     def end(self):
-        end = super(AllDayEvent, self).end
+        end = super().end
         if end == self.start:
             # https://github.com/pimutils/khal/issues/129
             logger.warning('{} ("{}"): The event\'s end date property '
```

Comment:
what about this one? .format methods in log strings are executed anyway, I dont think it makes much difference between using f-strings and .format.

Reply:
I didn't want to look until those changes were merged, but it looks like there are a lot (~90 on a quick grep) of instances of `'{}'.format(x)` that we could change if we want to go the whole hog. It would make it more consistent... I'll try to make a start in a bit. Also fixing style.

Reply:
I think that's all of them. If they seem excessive feel free to revert them!

Reply:
I think these ones are missing because of how the string is split across multiple lines.

Those might have to be looked at manually -- unless you're feeling brave enough to poke into `pyupgrade`.

Reply:
Re style, I'd just add a pre-commit hook with `black` and `reorder_python_imports`, and forget about dealing with that manually.

Reply:
> Those might have to be looked at manually -- unless you're feeling brave enough to poke into `pyupgrade`.

I did a `git grep {}` then looked at them manually. The style fixes were only because I was using GitHub suggestions rather than editing locally :)